docs/source/usage.rst

```diff
@@ -71,3 +71,35 @@ The format is JSON, with a single array containing each todo as a single entry
 given field, it will be printed as ``null``.
 
 Fields MAY be added in future, but will never be removed.
+
+Sorting
+-------
+
+The tasks can be sorted with the --sort argument. Sorting may be done according to the following fields:
+
+- STRING_FIELDS
+    * description
+    * location
+    * status
+    * summary
+    * uid
+    * rrule
+
+- INT_FIELDS
```

Comment:
I don't think it's necessary to have those categories, they're just for internal use.

Reply:
I just meant that you don't have to separate the attributes into `INT_FIELDS`, `STRING_FIELDS`, `DATETIME_FIELDS` etc because it doesn't make a difference to the user for this usecase.

There's nothing wrong with sorting by `uid`... I think todoman doesn't fail silently but actually tries to sort by the value of the UID property. If it makes sense to do that is a different thing, but who are we to judge the user :)

Reply:
Yes, indeed we cannot judge the user! ;-)

But, I think todoman is failing silently if we try and sort by `uid`.
Here's the output of `todo list --sort uid` on my machine:
```
42  [ ]  !!   07-07-2017 00:37  ---------------------
36  [ ]       23-06-2017 15:29  ---------------------
54  [ ]  !    21-11-2017 13:52  ---------------------
46  [ ]  !    30-06-2017 23:44  ---------------------
53  [ ]  !!!  23-10-2017 13:45  ---------------------
```
I also tried something similar with `location` (`STRING_FIELDS`), I made three todos whose location started with `a`, `b` and `c`, after sorting they were displayed in the following order: `b`, `a` and `c`.     ¯\_(ツ)_/¯

I haven't tried `LIST_FIELDS` and `RRULE_FIELDS` because I'm not sure of how todoman should ideally behave?

Reply:
I can reproduce this bug, let's discuss this in a separate issue.

Reply:
https://github.com/pimutils/todoman/issues/295